packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -319,15 +319,17 @@ class Slider extends React.Component {
   };
 
   emitChange(event, rawValue, callback) {
-    const { step, value: previousValue, onChange, disabled } = this.props;
+    const { step, value: previousValue, onChange, disabled, min, max } = this.props;
     let value = rawValue;
 
     if (disabled) {
       return;
     }
 
     if (step) {
-      value = roundToStep(rawValue, step);
+      if (!(rawValue <= min || rawValue >= max)) {
```

Comment:
This is equivalent to `rawValue > min && rawValue < max` which is easier to understand.